web/src/lib/components/album-page/albums-list.svelte

```diff
@@ -257,8 +256,13 @@
     await deleteSelectedAlbum();
   };
 
-  const handleEdit = (album: AlbumResponseDto) => {
-    albumToEdit = album;
+  const handleEdit = async (album: AlbumResponseDto) => {
+    const editedAlbum = await modalManager.show(EditAlbumForm, {
+      album,
+    });
+    if (editedAlbum) {
+      successEditAlbumInfo(editedAlbum);
+    }
     closeAlbumContextMenu();
```

Comment:
Do you want to close the context menu before the modal is opened? That's how it was before. (kind of, because before the operation before wasn't blocking)

Reply:
Can probably revisit this eventually. For now I don't want to risk running into some random edge cases on some browser due to an unrelated refactor :)